ui/components/app/asset-list/asset-list.js

```diff
@@ -87,11 +87,11 @@ const AssetList = ({ onClickAsset }) => {
         onTokenClick={(tokenAddress) => {
           onClickAsset(tokenAddress);
           trackEvent({
-            event: 'Clicked Token',
+            event: EVENT_NAMES.TOKEN_SCREEN_OPENED,
             category: EVENT.CATEGORIES.NAVIGATION,
             properties: {
-              action: 'Token Menu',
-              legacy_event: true,
+              token_symbol: primaryCurrencyProperties.suffix,
+              location: 'Home',
```

Comment:
Once you update your branch with the latest code from `develop`, you could do this when you need to set the `location` field based on a URL: https://github.com/MetaMask/metamask-extension/blob/develop/ui/pages/swaps/awaiting-swap/awaiting-swap.js#L173

That way if the 'Home' string (or other page names based on URLs) would change in the future, it would only need to be updated in one place: https://github.com/MetaMask/metamask-extension/blob/develop/ui/helpers/constants/routes.js#L106

However, if you want to track different components on the same page, you could just add a more specific string directly into the `location` property or rely on the combination of category + location loaded from context.page.title.

Reply:
> From [awaiting-swap.js#L173](https://github.com/MetaMask/metamask-extension/blob/develop/ui/pages/swaps/awaiting-swap/awaiting-swap.js#L173): `contextPropsIntoEventProperties: [CONTEXT_PROPS.PAGE_TITLE]` 

context props, as outlined here, aren't automatically passed through.  i'd prefer to keep as-is (hardcoded text) for now and revisit later as a separate ticket (as this occurs in many places)